src/c++/perf_analyzer/genai-perf/genai_perf/llm_inputs/shared.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Not a fan of the filename....but, I can't think of something that's more descriptive at the moment.

Reply:
maybe `common.py` or `utils.py`?

Reply:
This is my comment too. Can we move this to something like inputs_shared_utilities or something like that?

Reply:
Renamed to` inputs_utils.py` for now. If folks prefer and agree to a different name later, happy to update in another PR for this feature branch (or we can always update it later as well).